proto/beacon/db/powchain.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package prysm.beacon.db;
+
+import "eth/v1alpha1/beacon_block.proto";
+
+option go_package = "github.com/prysmaticlabs/prysm/proto/beacon/db";
+
+// ETH1ChainData is a container which holds all the relevant eth1
+// information
+message ETH1ChainData {
+    LatestETH1Data current_eth1_data = 1 ;
+    ChainStartData chainstart_data = 2;
+    SparseMerkleTrie trie = 3;
+}
+
+// LatestETH1Data contains the current state of the eth1 chain.
+message LatestETH1Data {
+    uint64 block_height = 2;
+    uint64 block_time = 3;
+    repeated bytes block_hash = 4;
```

Comment:
do we need ssz-size tag for this?

Reply:
we do not need it, we are just using protobufs to serialize this for disk storage.